static/js/drafts.js

```diff
@@ -133,6 +133,13 @@ exports.restore_draft = function (draft_id) {
     if (draft.type === "stream" && draft.stream === "") {
         draft_copy.subject = "";
     }
+    if (draft.type === "stream") {
+        narrow.activate([{operand: draft_copy.stream, operator: "stream"},
+                         {operand: draft_copy.subject, operator: "topic"}]);
+    }
+    if (draft.type === "private") {
+        narrow.activate([{operand: draft_copy.reply_to, operator: "pm-with"}]);
+    }
```

Comment:
OK, and when you mean "invalid recipients," that just means that there does not exist such stream and/or topic and/or person?